examples/client_async_test.cpp

```diff
@@ -2,87 +2,88 @@
 #include <iostream>
 
 #include <msp_msg.hpp>
-#include <msg_print.hpp>
+//#include <msg_print.hpp>
 
 struct SubCallbacks {
-    void onIdent(const msp::msg::Ident& ident) {
+    void onIdent(msp::msg::Ident& ident) {
```

Comment:
Why remove the `const`? Since you are providing a reference, it should be `const` so the user doesn't change it.